packages/cxl-ui/src/components/cxl-popover-card.js

```diff
@@ -0,0 +1,193 @@
+  }
+
+  async firstUpdated() {
+    await super.firstUpdated();
+  }
+
+  async _triggerMouseOver(event) {
+    if (this.timeoutId) {
+      clearTimeout(this.timeoutId);
+    }
+
+    this._getAnchorPosition(event.target);
+
+    this._removeTemporaryElement();
+
+    const playbookObj = await this._fetchData(this.apiUrl);
+
+    const content = await this._prepareContent(playbookObj);
+
+    await this._showTemporaryElement(content);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  async _triggerMouseOut() {
+    if (this.timeoutId) {
+      clearTimeout(this.timeoutId);
+    }
+
+    this.timeoutId = setTimeout(() => {
+      this._removeTemporaryElement();
+    }, this.timeoutDelayMs);
+  }
+
+  _getAnchorPosition(anchor) {
+    const rect = anchor.getBoundingClientRect();
+
+    const offsets = this._getOffset(anchor);
+
+    this.anchorPosition = {
+      width: anchor.offsetWidth,
+      height: anchor.offsetHeight,
+      top: offsets.top,
+      left: offsets.left,
+      viewportTop: rect.top,
+      viewportLeft: rect.left,
+      viewportBottom: rect.bottom,
+      viewportRight: rect.right,
+    };
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  _removeTemporaryElement() {
+    const el = document.getElementById('cxl-popover-tmp-div');
+    if (el) {
+      el.remove();
+    }
+  }
+
+  _getAnchorPositionTop() {
+    return this.anchorPosition.top - 50;
+  }
+
+  _getAnchorPositionLeft() {
+    return this.anchorPosition.left + this.anchorPosition.width + 50;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  _getOffset(el) {
+    let _x = 0;
+    let _y = 0;
+    while (el && !isNaN(el.offsetLeft) && !isNaN(el.offsetTop)) {
+      _x += el.offsetLeft - el.scrollLeft;
+      _y += el.offsetTop - el.scrollTop;
+      // eslint-disable-next-line no-param-reassign
+      el = el.offsetParent;
+    }
+    return { top: _y, left: _x };
+  }
+
+  /**
+   * @param {TemplateResult<1>} content
+   * @return {Promise<void>}
+   * @private
+   */
+  // eslint-disable-next-line class-methods-use-this
+  async _showTemporaryElement(content) {
+    const el = document.createElement('div');
+
+    el.id = 'cxl-popover-tmp-div';
+    el.style.position = 'absolute';
+    el.style.left = `${this._getAnchorPositionLeft()}px`;
+    el.style.top = `${this._getAnchorPositionTop()}px`;
+    el.style.maxWidth = 'calc(var(--cxl-content-width) * 0.7)';
+    el.style.zIndex = 1000;
+
+    el.onmouseenter = () => {
+      if (this.timeoutId) {
+        clearTimeout(this.timeoutId);
+      }
+    };
+
+    el.onmouseleave = () => {
+      this.timeoutId = setTimeout(() => {
+        this._removeTemporaryElement();
+      }, this.timeoutDelayMs);
+    };
+
+    render(content, el);
+
+    document.body.appendChild(el);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  async _prepareContent(el) {
+    return html`
+      <cxl-vaadin-accordion
+        id="cxl-vaadin-accordion-26107"
```

Comment:
hardcoded `id` also looks weird

Reply:
@pawelkmpt right, this can be related to `cxl-popover-card`s id, I'll change this.

Reply:
@pawelkmpt  fixed here https://github.com/conversionxl/aybolit/pull/169/commits/fd104e2054e09bcf02e202b6002163bc53c9d759